docs/rst/tutorials/cloud/debug_ros2_router/debug_ros2_router.rst

```diff
@@ -0,0 +1,248 @@
+At this point, the ROS 2 Router will be deployed as a bridge between the two Domains and will enable the ``talker``-``listener`` communication.
+Please take into account that a specific configuration will be applied to the ROS 2 Router in order to see its status and operation at runtime.
+
+.. warning::
+
+    It is important to mention that the performance of the ROS 2 Router will be affected due to the generation of extra code to present this information so it is not recommended to compile the *ROS 2 Router* following this tutorial in critical or production environments.
+
+
+Prerequisites
+-------------
+
+It is required to have previously installed Vulcanexus using one of the following installation methods:
+
+* :ref:`linux_binary_installation`
+* :ref:`linux_source_installation`
+* :ref:`docker_installation`
+
+Deploy ROS 2 nodes
+------------------
+
+First let's run the ROS 2 ``talker`` and ``listener`` nodes.
+
+Environment setup
+^^^^^^^^^^^^^^^^^
+
+Setup the Vulcanexus environment to have the ``demo_nodes_cpp`` package available.
+For this, there are two possible options:
+
+#.  Running the Vulcanexus Docker image.
+
+    Run the Vulcanexus Docker image with:
+
+    .. code-block:: bash
+
+        docker run -it ubuntu-vulcanexus:humble-desktop
+
+    Then, within the container, source the Vulcanexus installation with:
+
+    .. code-block:: bash
+
+            source /opt/vulcanexus/humble/setup.bash
+
+#.  Setting up the development environment on the local host.
+    For this second option, it is necessary to have installed the ``vucanexus-humble-desktop`` package, since this is the one that includes all the simulation tools, demos and tutorials.
+
+    Source the following file to setup the Vulcanexus environment:
+
+    .. code-block:: bash
+
+            source /opt/vulcanexus/humble/setup.bash
+
+Running ROS 2 nodes
+^^^^^^^^^^^^^^^^^^^
+
+Once the environment has been setup using one of the above options, run the ROS 2 ``talker`` node in one terminal.
+
+.. code-block:: bash
+
+    ros2 run demo_nodes_cpp talker
+
+Then, on another terminal, run the ROS 2 ``listener`` node in ROS 2 Domain ``1``.
+
+.. code-block:: bash
+
+    ROS_DOMAIN_ID=1 ros2 run demo_nodes_cpp listener
+
+
+At this point, the ``listener`` should not receive any data.
+If not, please go again through the previous steps.
+
+
+Deploy ROS 2 Router
+-------------------
+
+Then, create the ROS 2 Router configuration file as the one shown below.
+
+.. note::
+
+    If deploying Vulcanexus from the Docker image, note that you will need to have a configuration file (config.yaml) for the DDS Router Edge accessible from your Docker container.
+
+    This can be achieved by mounting a shared volume when launching the container, by copying the file from the local host to the container in case it is already running, or by editing a file from the Docker container itself.
+
+.. literalinclude:: /resources/tutorials/cloud/change_domain/change_domain_with_echo.yaml
```

Comment:
NIT: add a button to download the yaml file?

Reply:
I'm not sure if we can do that

Reply:
Kind of in the same line of thought: https://github.com/eProsima/vulcanexus/pull/86#discussion_r1018947827

Reply:
It is possible to do it explicitly: https://fast-dds.docs.eprosima.com/en/latest/fastdds/ros2/discovery_server/ros2_discovery_server.html#compare-discovery-server-with-simple-discovery
However I am not sure we would want that